pkg/graveler/ref/manager.go

```diff
@@ -86,7 +86,11 @@ func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.Re
 				INSERT INTO graveler_branches (repository_id, id, staging_token, commit_id)
 				VALUES ($1, $2, $3, $4)`,
 			repositoryID, repository.DefaultBranchID, token, commitID)
+
 		if err != nil {
+			if errors.Is(err, db.ErrAlreadyExists) {
+				return nil, graveler.ErrNotUnique
```

Comment:
We rely on graveler to now return ErrAlreadyExists for these methods - I think the interface should document that (i.e. an implementation of it is now expected to return this error for duplicate repos, branches, tags).
Go makes this pretty hard to express in code, so let's at least document it.

Reply:
Done.